waspc/data/Generator/templates/sdk/wasp/client/operations/queries/index.ts

```diff
@@ -12,5 +12,5 @@ export const {= operationName =}: QueryFor<{= operationTypeName =}> = createQuer
 )
 {=/ queries =}
 
-// PRIVATE API
-export { addMetadataToQuery } from './core'
+// PRIVATE API (used in SDK)
```

Comment:
If used only in SDK, do you need to expose in the `package.json`?

Reply:
No, that's what we'll do as part of #1922.

But for now, yes (since other parts of SDK use it as a package import).